docs/remote-interface.md

```diff
@@ -77,3 +77,17 @@ $ make BUILD_REMOTE=1 BUILD_DEBUG=1 check
 
 ## Requirements
 Remote interface requires RabbitMQ broker
+
+## Fallback mechanism
+If the remote interface is not working for some reason, the PoW calculation will be transfer to the local hardwares.\
```

Comment:
Fix the grammar. "be transfer" -> "be transferred"

Reply:
Fixed.